src/controllers/controller.js

```diff
@@ -378,6 +379,36 @@ class Controller extends EventEmitter {
 		return addr
 	}
 
+	async getIntersection(latitude, longitude) {
+		if (this.config.geocoding.intersection_users == [""]){
```

Comment:
needs updating for camel case
should also live through not being present.
perhaps
`
if (!this.config.geocoding.intersectionUsers || !this.config.geocoding.intersectionUsers.length) ?